### ISSUE TITLE
Jetpack plugin details page won't load

### ISSUE THREAD

**Issue by alisterscott** [Automattic/wp-calypso#11184]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/plugins/yourjetpacksite.com
2. Click a plugin

#### What I expected

To see plugins detail page

#### What happened instead

Nothing 

Console log

```
Warning: Failed prop type: The prop `store` is marked as required in `Provider`, but its value is `undefined`.
    in Provider
printWarning @ vendor.development.js:3350
2017-02-06 12:17:27.477 vendor.development.js:3305 Uncaught Error: _registerComponent(...): Target container is not a DOM element.
    at invariant (vendor.development.js:3298)
    at Object._renderNewRootComponent (vendor.development.js:21542)
    at Object._renderSubtreeIntoContainer (vendor.development.js:21632)
    at Object.render (vendor.development.js:21653)
    at renderWithReduxStore (app:///./client/lib/react-helpers/index.js:64)
    at renderPage (app:///./client/lib/react-helpers/index.js:46)
    at renderSinglePlugin (app:///./client/my-sites/plugins/controller.js:114)
    at plugin (app:///./client/my-sites/plugins/controller.js:243)
    at vendor.development.js:30533
    at nextEnter (vendor.development.js:30365)
2017-02-06 12:17:32.018 manualtesting.mystagingwebsite.com Failed to load resource: the server responded with a status of 500 (Internal Server Error)
```

#### Browser / OS version

Chrome OSX

#### Context / Source
 #automated-testing

**Comment by alisterscott** [comment]
This was introduced in https://github.com/Automattic/wp-calypso/pull/10590

**Comment by alisterscott** [comment]
Closed via #11185
